### DISCUSSION TITLE
Spinning up the JetBot

### DISCUSSION THREAD

**Discussion by Scar3cr0w101** [NVIDIA-AI-IOT/jetbot#393] [Q&A]
Hey all, 

I have been on a bit of a rollercoaster with my Jetson! I am an experienced linux user and very familiar with single board computers. I initially bought my jetson for a couple of non AI/robotics related projects (after trying on a pi3, then getting a pi4, realising it doesnt have the decoders for >1080p, picking up a jetson, realising it has no audio output and finally getting to a workable state with an inline HMDI splitter...!) and thought I would grab a jetbot kit (waveshare) to build something with my children over half term. 

Waveshare... my oh my, useless instructions! I had to use three different mediums and sets of instructions to piece something thogether that looked vaguely like the picture (:D). I also didnt realise the nano 2gb didnt have the wifi chip slot nor the fan pins. So they are dormant. Ive also now got a 2 - 3 pin adapter so I can charge the batteries. 

I have flashed a 64gb SD (that was running the Nvidia OS previously and worked fine) with this image :

jetbot-042_nano-2gb-jp441.zip

No dice. I have unplugged the motor board, inserted the SD card, attached peripherals and powered the Jetson with the USB-C power input. But I dont get a video signal. I have re flashed the card several times, used initdisk to put the card in as clean a state as possible, flashed it again (all using etcher) but still no joy! Im starting to get to the point where I am going to start ploughing through the manual build instructions but I thought I would ask for some help here first! Im looking forward to getting started.

Any help would be greatly appreciated!

Pete

**Comment by culebracut** [comment]
you may need the jumpered power supply instead of the usb as it can produce more power.

**Reply by Scar3cr0w101** [reply]
But, surely it should be able to function off of the batteries alone? I'm only asking it to boot the Jetson, not the control board etc.

**Reply by rob-opsi** [reply]
have you fitted the jumper to switch off usb power, so you can use either jumper cables to the batteries, or a barrel jack power supply?

**Reply by Scar3cr0w101** [reply]
I haven't no. Not to my knowledge. I have tried powering it on with both the battery Baird connected and disconnected

**Comment by jaybdub** [comment]
Hi @Scar3cr0w101 ,

Thanks for reaching out, and apologies for the delayed response!

One thing to first check might be that official Jetson Nano SD card image is working on your platform.  

https://developer.nvidia.com/embedded/jetpack

If you can boot, and use the system as normal, this might help eliminate a variety of potential issues.  After that is launched, it may then be possible to use the JetBot docker image directly to enable a JetBot configuration.  

Sorry for the frustration you've been experiencing.  

Please let me know if this helps or you run into issues.

Best,
John

**Comment by santaimpersonator** [comment]
Try the plain jetpack 4.5 image, as noted in the call out at the bottom of **Step1** here: https://jetbot.org/master/software_setup/sd_card.html

If you are using a custom image with Jetpack 4.5 the board doesn't do anything.
